aws-operator-role/iam-policy.json

```diff
@@ -61,7 +61,8 @@
                 "sts:GetFederationToken",
                 "servicequotas:*",
                 "support:*",
-                "trustedadvisor:*"
+                "trustedadvisor:*",
+		"sqs:*"
```

Comment:
If we can we should try and make this smaller. Customers will likely not like this